content/change/teaching/resources.md

```diff
@@ -1,13 +1,16 @@
-title: Teaching resources
+title: Resources
 
 # Where can I find resources for teaching open science?
 
 Open Science is here to stay. Therefore, students deserve to be prepared for a future where reproducable research practices are standard. We have found that students intuitively understand the necessity for changes to the scientific practice, and enjoy applying state-of-the-art methods in their projects.
 
 As with scientific resources, members of the Open Science community have created and share freely their teaching materials. Here are a few resources:
 
-* The [**Open Science Training Handbook**](https://book.fosteropenscience.eu) is a fantastic and comprehensive resource that covers many topics around open science. It's built for people who would like to create open science training, and very in-depth.
 * Many researchers have made their [**course syllabi for open and reproducible methods**](https://osf.io/vkhbt/) openly available.
+* At the SIPS Meeting 2018, a Hackathon collected a whole bunch of [teaching resources](https://osf.io/bts5d/wiki/home/), including syllabi, videos, podcasts, journal articles and much more.
+* The [Framework for Open and Reproducible Research Training](https://forrt.netlify.com) presents recommendations to improve the adoption of open science teaching practices by modifying the academic incentive structure.
```

Comment:
Nice! I haven't looked at this in a lot of detail, but I'm wondering whether it would make sense to link to their [syllabus](https://forrt.netlify.com/as.resource.tool.html) and [principles](https://forrt.netlify.com/principles.html) directly?